helm/sematic-server/templates/deployment.yaml

```diff
@@ -33,6 +52,9 @@ spec:
             {{- toYaml .Values.deployment.container_security_context | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pull_policy }}
+          env:
+          - name: SEMATIC_WSGI_WORKERS_COUNT
+            value: "{{ .Values.deployment.socket_io.worker_count }}"
```

Comment:
This must be deactivated for the socketio instance.

Reply:
this deployment file is for the API instance, so this is fine. We just shouldn't put it in the configmap

Reply:
yes, normally we avoid specifying env vars directly like this, but i did this one as an exception because it's likely to be the only env difference between the api/socketio instances.  it is not specified for the socketio instance.